trainings/batch-changes-webinar/lesson-plan.md

```diff
@@ -0,0 +1,125 @@
+
+
+# Batch Changes Webinar
+
+**Topic:** This training is intended as an initial deep-dive to educate users on Batch Changes outside of the 101/102 context. If possible, we recommend having the customer provide a batch change use case to you *before* the training, giving you the opportunity to work on that specific change or something similar; if not, you can use this as a way to show Batch Changes to them and discuss use cases they might find valuable live on the training.
+
+## Intro
+
+Today's training is intended to walk users through our Batch Changes product, and show users how to use the feature. By the end of it, we hop that you'll feel comfortable with what Batch Changes is and can do, and will be ready to start with your own changes in Sourcegraph.
+
+❗️ If the training is happening in the context of a champion brining in other team members, have them speak at the top about why they think Batch Changes will be useful, and what they envision the team using it for.
+
+## Unit 1: Get familiar with Batch Changes
+
+**Learning goals:** After completing this unit, customers will understand what Batch Changes is.
+
+### What is Batch Changes?
+
+Batch Changes is a way to script changes to your code. Using Sourcegraph search, you'll find a list of relevant repositories—for example, all repos that have `Dockerfile`s, or all repos that have Java files. Then, using a YAML spec that you write, you'll script out the change that you want to make to your files. We can support something as simple as a find and replace (essentially `sed` on steroids) or as complex as a full refactor. Batch Changes can run across many repos at once, allowing you to make very large-scale changes—particularly helpful for the sorts of rote activities (like adding a new step to the CI pipeline) that may need to happen at scale.
+
+Once the Batch Changes spec is executed, Sourcegraph will generate pull/merge requests on your code host, one per repo (usually—this can be customized). We'll open them on your behalf, and then give you the ability to manage them from within Sourcegraph. You can close individual PRs, comment on them, and even bulk-merge. We also generate a burndown chart for you to track progress. 
+
+### Running a Batch Change
+
+To get started, I'm going to run a Batch Change that appends `Hello World` to all my `README` files.
```

Comment:
nit: we usually capitalize only the product name, not the entity (so, you can use Batch Changes to create batch changes).

Reply:
Hahah, I was wondering what the usual approach was. I'll fix it.

Reply:
Done!